packages/graphql-dynamodb-transformer/src/definitions.ts

```diff
@@ -539,8 +544,15 @@ export function makeModelConnectionField(fieldName: string, returnTypeName: stri
         makeInputValueDefinition('nextToken', makeNamedType('String'))
     ];
     if (sortKeyInfo) {
+        let namedType : NamedTypeNode;
+        if (sortKeyInfo.typeName === 'Composite') {
```

Comment:
Can we create a ```type SortKeyInfo = 'Composite' | string``` Type to constrain the valid values of string literals because we're using ```Composite``` at multiple places and it would give some TypeScript safety and would say more about the type that it is a special value.